### DISCUSSION TITLE
Ingame HUD

### DISCUSSION THREAD

**Discussion by arc1115** [ejektaflex/Bountiful#191] [Mod Ideas]
### Feature Description

an ingame hud after adding a bounty to the inventory that shows what bounties are active. 

### Feature Benefits

this will make it more immersive in my opinion as a questing mod and help with ppl like me who sometimes forget they dont have a bounty active in their inventory

**Comment by ejektaflex** [comment]
This is a great idea. The ability to make a bounty in your inventory as "active" would be quite nice.

**Comment by ejektaflex** [comment]
I haven't gotten around to this yet. However, I have added notifications that fire when you complete bounties!
